e2e/cypress/integration/notifications/direct_messages_do_not_add_indicator.js

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Please add `_spec` to filename so it'll be [included for testing](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json#L9).